### DISCUSSION TITLE
New web site for translators

### DISCUSSION THREAD

**Discussion by kierepka** [Megabit/Blazorise#1640] [Show and tell]
We create a system for translators in 2 people.
UI application in the form of WebAssembly made in C # using blazorise, fluxor and a few other additions.
Codes here: [OriinDictionary](https://github.com/kierepka/OriinDictionary), direct link to app: [Oriin Dictionary](https://oriin.ga/)

When I finish the application, I would like to add:
1) Virtual scroll for the grid based on the MatBlazor codes:
[MatVirtualScroll ](https://github.com/SamProf/MatBlazor/tree/master/src/MatBlazor/Components/MatVirtualScroll )- maybe it will come to the official repository later?
2) Improve the overall css for Bulma - there are still some problems

Thank you for this library and happy holidays and a successful new year.

**Comment by stsrki** [comment]
Hi, I tried to look at your project but it's giving me 404 errors.

Regarding your suggestions:

1. virtual scroll seems like a nice idea. From what I can see it's the same as `Virtualize` from native Blazor. https://docs.microsoft.com/en-us/aspnet/core/blazor/components/virtualization?view=aspnetcore-5.0
2. Any help with Bulma is appreciated :)

Happy holidays to you too :)

**Reply by kierepka** [reply]
Hi @stsrki,
Ad 1) No it's something like [Lazy loading in pure JavaScript](https://www.merixstudio.com/blog/lazy-loading-pure-javascript/ )
Ad 2) Great I'm in
Ad 404 - hmm can you give me screenshot? I tested on my computer and using: 
a) https://en.ryte.com/website-checker/
b) https://gtmetrix.com/reports/www.oriin.ga/FR9yKJBV/
c) https://sitechecker.pro/app/main/project/1587151/audit/summary 
No 404 errors :(

**Comment by stsrki** [comment]
This is the error

![image](https://user-images.githubusercontent.com/900302/103002506-519daf00-452f-11eb-847d-0be458c7df06.png)

**Comment by stsrki** [comment]
PS. lazy load virtual scroll could be great as a Blazorise extension :)

**Comment by kierepka** [comment]
Ha you have right - problem with 1 link after my update. 

I will fix this: request https://oriin.ga:8000/api/v1/languages/?page=**0**&per_page=50 to https://oriin.ga:8000/api/v1/languages/?page=**1**&per_page=50

**Reply by kierepka** [reply]
Fix and UI changes already on website (CTRL+F5)

**Comment by stsrki** [comment]
This is really strange, but I still get the errors

![image](https://user-images.githubusercontent.com/900302/103338995-7f7b7a00-4a80-11eb-852e-8a0c699c8072.png)

**Comment by kierepka** [comment]
On [stackoverflow there is thread](https://stackoverflow.com/questions/23341765/getting-neterr-blocked-by-client-error-on-some-ajax-calls). This is your client - so probably adblock or opera blocker?
Other links:
1) https://love2dev.com/blog/what-the-heck-does-neterr_blocked_by_client-mean-and-how-can-you-fix-it/
2) https://www.keycdn.com/support/how-to-solve-err-blocked-by-client (whitelist domain)

I checked it on several computers (opera/chrome/edge/IE/firefox) without adblock and no errors like this.

**Comment by stsrki** [comment]
Why would I need to disable AdBlock to visit a webpage? Seems fishy from user's perspective.

**Reply by kierepka** [reply]
But this adblock is blocking main app 👎

**Comment by stsrki** [comment]
I disabled AdBlock for now and now I'm able to see the app. it looks really nice I must say :) You also did a theme change, cool.

Regarding the AdBlock, you should consider removing the scripts or files that are blocking the app from loading.

**Reply by kierepka** [reply]
@stsrki I just send this as error to aspnetcore team: https://github.com/dotnet/aspnetcore/issues/28916

**Reply by kierepka** [reply]
OK - new UI redesign based on new Blazorise layout system - you can check [oriin](https://oriin.ga)

**Comment by stsrki** [comment]
@kierepka Can you please explain more about the application? What is it used for and what is the benefit of it? Why is it different from a built-in IStringLocalizer, for example?
